network/router.go

```diff
@@ -199,6 +205,13 @@ func (r *Router) removeConnection(si *ServerIdentity, c Conn) {
 	r.connections[si.ID] = arr[:len(arr)-1]
 }
 
+// triggerConnectionErrorHandlers trigger all registered connectionsErrorHandlers
+func (r *Router) triggerConnectionErrorHandlers(remote *ServerIdentity) {
```

Comment:
I think `connectionErrorHandlers` is the correct way of writing it, while `connectionsErrorHandler` puts the plural at the wrong place. So:
can you rename `connectionsErrorHandler` to `connectionErrorHandlers`?